css/_components.scss

```diff
@@ -51,3 +51,69 @@
         }
     }
 }
+
+
+
+/**
+ * The background-video mixin styles an edge-to-edge background video. Because 
+ * elements within .hero are absolutely positioned, .hero’s height gets wiped 
+ * out. We assign it a faux-height via a ::before pseudo element.
+ *
+ * Emmet: .background-video>.hero>.panel{$1}+video
+ */
+@mixin background-video() {
+    .container {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 1140px;
+        padding-left: 20px;
+        padding-right: 20px;
+        position: relative;
+        z-index: 2;
+    }
+
+    .hero {
+        overflow: hidden;
+        padding: 40px 0;
+        position: relative;
+        .panel {
+            left: 0;
+            position: absolute;
+            top: 0;
+            width: 100%;
+            z-index: 3;
+            > div {
+                color: #FFF;
+                margin-left: auto;
+                margin-right: auto;
+                margin-top: 30px;
+                max-width: 1140px;
+                padding-left: 20px;
+                padding-right: 20px;
```

Comment:
I'd probably remove the stuff equivalent to .container above, and just use class="container" on this div in the example. That way, the component isn't dependent on using .container, but you can demonstrate that it can be used with container if you'd like.